display_list/dl_canvas.h

```diff
@@ -118,7 +118,7 @@ class DlCanvas {
   virtual void ClipPath(const DlPath& path,
                         ClipOp clip_op = ClipOp::kIntersect,
                         bool is_aa = false) {
-    ClipPath(path.GetSkPath(), clip_op, is_aa);
+    ClipPath(path.GetSkPath(true), clip_op, is_aa);
```

Comment:
minor nit. For these sorts of booleans, I like the argument comments `/*will_render=*/true

Reply:
Or use a second method with a more obvious name?

Reply:
if its just used in a few places the comment is probably fine, but your choice.